src/main/java/org/broadinstitute/hellbender/utils/IntervalUtils.java

```diff
@@ -1293,7 +1298,7 @@ static public int endOfShard(int shardIndex, int shardSize) {
         final List<T> sortedLocatables1 = sortLocatablesBySequenceDictionary(locatables1, dictionary);
         final List<T> sortedLocatables2 = sortLocatablesBySequenceDictionary(locatables2, dictionary);
 
-        return combineBreakpoints(sortedLocatables1, sortedLocatables2);
+        return  sortLocatablesBySequenceDictionary(combineBreakpoints(sortedLocatables1, sortedLocatables2), dictionary);
```

Comment:
I actually think that just one method named `combineAndSortBreakpoints` that always returns a sorted result (even if only a single list is passed) would suffice.  I can't see a scenario in which you'd want a list without a guaranteed order?

Also, is sorting the two lists of locatables redundant?

Reply:
Not redundant.  Since the combineBreakpoints makes assumptions about sorting.

Done

Reply:
I removed this method and replaced it everywhere with your `combineAndSortBreakpoints`, and all tests passed.  Sorting the copies of the two input lists is pretty much redundant if you are going to sort the final list, right?